content/03.methods.md

```diff
@@ -138,18 +138,19 @@ For both these tools we used aligned BAM and chimeric SAM files from STAR as inp
 We ran STAR-Fusion with default parameters and annotated all fusion calls with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz provided in the STAR-fusion release. 
 For Arriba, we used a blacklist file (blacklist_hg38_GRCh38_2018-11-04.tsv.gz) from the Arriba release tarballs to remove recurrent fusion artifacts and transcripts present in healthy tissue.
 We also provided Arriba with strandedness information or set it to auto-detection for polyA samples.
-The RNA expression and fusion workflows can be found in the [KidsFirst GitHub repository](https://github.com/kids-first/kf-rnaseq-workflow).
+We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) to harmonize Arriba fusion annotations with those of STAR-Fusion.
+The RNA expression and fusion workflows can be found in the [KidsFirst GitHub repository](https://github.com/kids-first/kf-rnaseq-workflow) and the FusionAnnotator workflow found in the [D3b GitHub repository](https://github.com/d3b-center/FusionAnnotator).
 
 #### Fusion prioritization
 
-We built a [fusion prioritization pipeline](https://github.com/d3b-center/fusion_filtering_pipeline) to filter and annotate fusions.
-We considered all inframe and frameshift fusion calls with 1 or more junction reads and fused genes expressed with TPM greater than one to be true calls.
-If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten) or if either 5\` or 3\` genes fused to more than five different genes we removed these calls as a potential false positive.
-We also removed fusions if the 5\` or 3\` ends were the same gene, and these were tagged as non-canonical splicing or duplication.
-We used a list of curated fusion calls for each histology to capture each occurrence of the fusion as a putative driver fusion.
-We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent in histology (>2 samples) or the fusion was specific to the broad histology. 
-We annotated putative driver fusions and prioritized fusions lists with kinases, oncogenic, tumor suppressor, transcription factor, fused genes and known TCGA fusions from curated [datasheets](https://github.com/d3b-center/fusion_filtering_pipeline/tree/master/references).
-We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.
+We performed artifact filtering and additional annotation on fusion calls to prioritize putative oncogenic fusions.
+Briefly, we considered all inframe and frameshift fusion calls with 1 or more junction reads and fused genes expressed with TPM greater than one to be true calls.
+If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten).
```

Comment:
I think you are missing the second half of the sentence here.

Reply:
whoops, fixed, thank you!